fill roadmap

cwmeijer opened the pull request.

elboyran reviewed (commented):
@cwmeijer , nice summary of the plans. Some suggestions:

- Adding Embeddings and other data modalities under: Data modalities section?
- Shall we also add more XAI methods as a section and mention the ones I mention at the README? With the explicit statement that we welcome contributions and the link to the operations repo of the guy who had answered your request some time ago saying it's a work in progress?
- Shall we also address Faruk's suggestion to link plans to future releases or very rough time line? Or not yet?

cwmeijer commented:
- I wanted to mention the embeddings separately because it is different with respect to the delivery date and state of funding etc. I added a mention to it in the 'other modalities' section to make sure it isn't easily overlooked.
- I added the XAI methods from the readme to the roadmap.
- I don't feel we are organized enough to make a sensible estimation what will be on what release exactly.

Could you have another look and see if you are happy with the changes I made after your suggestions? 💁‍♂️

elboyran commented:
It's fine.
